File: PYTHON/account.py
class Account:
    id = int
    name = str
    document = str
    email = str
    password = str

    def __init__(self, name, document, email, password):
        self.name = name
        self.document = document
        self.email = email
        self.password = password

    def toString(self):
        return "Name: " + self.name + ", Document: " + self.document + ", Email: " + self.email + ", Password: " + self.password

File: JAVA/account.java
public class account {
    public int id;
    public String name;
    public String document;
    public String email;
    public String password;

    public account(String name, String document, String email, String password) {
        this.name = name;
        this.document = document;
        this.email = email;
        this.password = password;
    }

    @Override
    public String toString() {
        return ("Name: " + name + ", Document: " + document + ", Email: " + email + ", Password: " + password);
    }
}
